src/gtk/toga_gtk/widgets/base.py

```diff
@@ -58,7 +59,8 @@ def set_color(self, color):
         pass
 
     def set_background_color(self, color):
-        # By default, background color can't be changed
+        if color:
+            self.native.set_background_color(self, color)
         pass
```

Comment:
The pass is no longer required if the method has a body.

Reply:
👍 yes.